tests/clar.c

```diff
@@ -313,11 +313,18 @@ clar_parse_args(int argc, char **argv)
 {
 	int i;
 
+	// Verify options before execute
```

Comment:
This is a nitpick, but would you mind changing this to C-style `/*` comments `*/`?  We don't use `//` within the project.

(I know - I can't imagine that anybody's using a compiler that doesn't accept `//` style comments these days _but_ we try to be generous about that stuff.)